src/status_im/contexts/wallet/address_input/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns status-im.contexts.wallet.address-input.view
```

Comment:
please consider where code lives. 
We have the wallet structured into large feature sets.

e.g 
status-im.contexts.wallet.home
status-im.contexts.wallet.send
status-im.contexts.wallet.account

imo this ns should be something like status-im.contexts.wallet.common.components.address-input.view